daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1562,6 +1562,26 @@ class RazerViperUltimateWireless(RazerViperUltimateWired):
     USB_PID = 0x007B
 
 
+class RazerViper(RazerViperUltimateWired):
```

Comment:
As those are two seperate devices, please don't inherit from it but just copy the properties over

Reply:
Indeed, those are two different devices. Fixed.